ush/rocoto/workflow_utils.py

```diff
@@ -339,6 +339,10 @@ def get_resources(machine, cfg, task, reservation, cdump='gdas'):
             natstr = "-l place=vscatter"
             if memory is None:
                natstr += ":exclhost"
+            natstr += ",debug=true"
+
+        if machine in ['WCOSS2'] and task in ['arch', 'earc', 'getic']:
+            natstr = "-l debug=true"
```

Comment:
You could move these 2 lines to 337 and drop the `and task in ['arch' ...]:`
Then change the line 339 `=` to `+=`
You won't need 342.

Reply:
I ingested your suggestion and further reduced that section from:
```
 if machine in ['WCOSS2'] and task not in ['arch', 'earc', 'getic']:
            natstr = "-l place=vscatter"
            if memory is None:
               natstr += ":exclhost"
            natstr += ",debug=true"

        if machine in ['WCOSS2'] and task in ['arch', 'earc', 'getic']:
            natstr = "-l debug=true"
```
to:
```
if machine in ['WCOSS2']:
    natstr = "-l debug=true"

    if task not in ['arch', 'earc', 'getic']:
       natstr += ",place=vscatter"
       if memory is None:
           natstr += ":exclhost"
```

The NATIVE entity in the rocoto xmls on WCOSS2 used to be (exclusive job example):
`-l place=vscatter:exclhost,debug=true`
...and are now:
`-l debug=true,place=vscatter:exclhost`
I have been running with this update on WCOSS2 all afternoon and it works. No observed submission errors. I have committed the adjustment to `ush/rocoto/workflow_utils.py` @ 0a21614.